### ISSUE TITLE
Autocomplete resets scroll to incorrect v. position

### ISSUE THREAD

**Issue by xixixao** [ajaxorg/ace#2540]
Open autocomplete
Scroll using arrows down enough to scroll the list
Complete
Open autocomplete again - the list is scrolled to the previous scrolled position

Now I don't mean Ctrl-Space open, I mean open using the API used in doLiveAutocomplete, and this probably only manifests when you're opening it without a prefix. What I can see is happening that the rendered on the popup calls its session to setScrollTop, but **after** that its VScrollBar resets it to what it was... Why the VScrollBar on "scroll" is getting triggered I have no clue...

**Comment by xixixao** [comment]
(my lame workaround is to do `this.popup.renderer.scrollBarV.setScrollTop 0` in detach of the popup)

**Comment by nightwing** [comment]
I couldn't reproduce this on chrome by doing
```js
editor.completer.showPopup(editor);
editor.completer.popup.navigateFileEnd();
editor.completer.detach()
editor.completer.showPopup(editor);
```

**Comment by xixixao** [comment]
Can you try using the keyboard instead of `editor.completer.popup.navigateFileEnd();`? I don't know if it makes a difference, but it might, as the culprit is the scroller and perhaps it is not engaged in the same way via the code.

**Comment by nightwing** [comment]
Keyboard doesn't seem to make any difference, maybe it depends on browser and os?
I have chrome on windows 7

**Comment by xixixao** [comment]
Forgot, sorry, OS X and FF.

Just checked Chrome and cannot repro, must be at least FF issue. Can you check FF on windows?

**Comment by nightwing** [comment]
Yes, I see this on FF on windows.
